modules/integration/src/test/scala/scala/cli/integration/CompileTestDefinitions.scala

```diff
@@ -57,40 +87,43 @@ abstract class CompileTestDefinitions(val scalaVersionOpt: Option[String])
     }
   }
 
-  test("test scope") {
+  test("copy compile output") {
     val inputs = TestInputs(
       Seq(
-        os.rel / "Main.scala" ->
-          """//> using lib "com.lihaoyi::utest:0.7.10"
-            |
-            |object Main {
-            |  val err = utest.compileError("pprint.log(2)")
-            |  def message = "Hello from " + "tests"
-            |  def main(args: Array[String]): Unit = {
-            |    println(message)
-            |    println(err)
-            |  }
-            |}
-            |""".stripMargin,
-        os.rel / "Tests.scala" ->
-          """//> using lib "com.lihaoyi::pprint:0.6.6"
-            |//> using target.scope "test"
-            |
-            |import utest._
-            |
-            |object Tests extends TestSuite {
-            |  val tests = Tests {
-            |    test("message") {
-            |      assert(Main.message.startsWith("Hello"))
-            |    }
-            |  }
-            |}
-            |""".stripMargin
+        mainInput,
+        testInput
+      )
+    )
+    val tempOutput = os.temp.dir()
```

Comment:
Let's avoid such temporary directories in tests. `root` can be assumed to be a temporary directory (where we write the inputs). A sub-directory of it would be fine for `--output`.

Reply:
done